app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,52 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  let lowerBoundNumber: string
+  let startingPointNumber: string
+  let upperBoundNumber: string
+
+  if (startingPoint !== null) {
+    startingPointNumber = formatBigNumber(startingPoint, 18, 2)
+  }
+
+  if (lowerBound !== null) {
+    lowerBoundNumber = formatBigNumber(lowerBound, 18, 2)
+  }
+
+  if (upperBound !== null) {
+    upperBoundNumber = formatBigNumber(upperBound, 18, 2)
+  }
```

Comment:
Since we now have access to the `BigNumber` return value of the `NumericalInput` component, we're actually better off to keep all the values as `BigNumber`s, so we can ditch all this